src/action_runner/action_runner.vcxproj

```diff
@@ -22,12 +22,11 @@
   <Target Name="GenerateResourceFiles" BeforeTargets="PrepareForBuild">
     <Exec Command="powershell -NonInteractive -executionpolicy Unrestricted $(SolutionDir)tools\build\convert-resx-to-rc.ps1 . resource.base.h resource.h action_runner.base.rc action_runner.rc" />
   </Target>
-
   <PropertyGroup Label="Globals">
     <VCProjectVersion>16.0</VCProjectVersion>
     <ProjectGuid>{D29DDD63-E2CF-4657-9FD5-2AEDE4257E5D}</ProjectGuid>
     <RootNamespace>actionrunner</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
```

Comment:
This change needs to be rolled back, the target platform cannot be changed.
This also applies to all projects.

Reply:
@enricogior Yes, I know that. It doesn't compile on my machine without upgrading the version, hence why you see it in my PRs. I will revert the changes before marking this PR ready to review